README.md

```diff
@@ -1,112 +1,89 @@
-9wm
-====
+## 9wm
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½, also known as rio.
-It provides a very simple and clean user interface.
-It is click-to-type.
-It uses the X11 font system (which, unfortunately, means no Unicode support).
+9wm is an X11 Window manager. It's inspired on the AT&T Plan 9 WM 8½, also known as [rio]. 
 
-9wm does not provide virtual desktops, customization, key bindings,
-EWMH support, or compositing.
-It does not allocate any colors (if you disable COLOR),
-which will be great news if you are stuck in 1993.
+It has four major features:
 
-It is a great place to start if you are interested in writing a window manager from scratch:
-many folks have done exactly this.
+- Simple and clean user interface. It is click-to-type.
+- It uses the X11 font system (which, unfortunately, means no Unicode support).
+- No virtual desktops, customization, key bindings, EWMH support, or compositing.
+-  It does not allocate any colors (if you disable COLOR), which will be great news if you are stuck in 1993.
 
-9wm is distributed under [an MIT License](LICENSE.md).
+It is a great place to start if you are interested in writing a window manager from scratch: many folks have done exactly this.
 
+## Install
 
-Where do I get it?
-==================
+Note: If you are on Arch Linux, you can use the [AUR]
 
-The latest version of 9wm is held at:
+We use [make] to build, install and setup `9wm`. So, in need to install:
 
-<https://github.com/9wm/9wm>
+```
+root # make install
+```
 
+## Usage
 
-How do I use it?
-================
+You need to include the follow line to your `.xinitrc`:
```

Comment:
Minor nitpick but I would include these instructions under the Install header, rather than usage, since these would be one-time installation instructions

Reply:
```suggestion
You can launch `9wm` through a display manager like `lightdm` or via `startx` directly.
To use `startx`, you'll need to include the following line at the end of your `.xinitrc`:
```